draft-ietf-quic-ack-frequency.md

```diff
@@ -486,7 +486,17 @@ new ACK_FREQUENCY frame immediately upon confirmation of connection migration.
 
 
 # Security Considerations
-TBD.
+
+An improperly configured or malicious sender could cause standards-compliant
+receivers to  acknowledge more frequently than their available resources permit,
+thus causing a denial-of-service (DoS) attack. However, a sender has control of
+many aspects of the receiver behaviour and therefore, for a well-configured
+sender, this introduces no additional threat. Further, the information in the
+ACK_FREQUENCY frame are ultimately only a request but do not provide any ability
+for the requester to enforce a certain behavior on the ACK sender. As such an
```

Comment:
It's a MUST earlier in the text though:
> On receiving an ACK_FREQUENCY frame and updating its recorded max_ack_delay and Ack-Eliciting Threshold values ({{multiple-frames}}), the endpoint MUST send an acknowledgement when one of the following conditions are met:
> 
> * Since the last acknowledgement was sent, the number of received ack-eliciting packets is greater than or equal to the recorded Ack-Eliciting Threshold.

Do we need to weaken that language a bit to allow the receiver more flexibility?

Reply:
I was wondering about this as well. I will open a separate issue!

Reply:
Issue #144